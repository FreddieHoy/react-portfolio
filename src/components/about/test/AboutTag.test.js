import { shallow } from 'enzyme';
import React from 'react';

import { AboutTitle } from '../AboutTitle';

describe('AboutTag', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<AboutTitle />);
    expect(wrapper).toMatchSnapshot();
  });
});
