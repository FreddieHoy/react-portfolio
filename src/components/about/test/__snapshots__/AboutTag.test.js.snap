// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutTag renders correctly 1`] = `
<AboutTitle__Container>
  <fontStyles__HeadingOne>
    This is more 
    <fontStyles__RedText>
      about 
    </fontStyles__RedText>
    me
    <fontStyles__BlueText>
      .
    </fontStyles__BlueText>
  </fontStyles__HeadingOne>
</AboutTitle__Container>
`;
